doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -270,6 +270,44 @@ Insufficiently restrictive variance annotations lead to errors:
 
    Fail Polymorphic Cumulative Record bad@{*a} := {p : Type@{a}}.
 
+.. example:: Demonstration of universe variances
+
+   Here is an example demonstating the behavior of various universe variances.
```

Comment:
```suggestion
.. example:: Behavior of various universe variances
```

With a good title, there's no need for an introductory sentence.
Since it's already labelled as an example, no need to use "Demonstration" or "Here is an example".

Reply:
By the way, should the other examples in this page also be `.. example`?

Reply:
IMHO longer examples and maybe some that cover more important points should be `.. example::`.  But overusing that (with the example header and extra spacing) can reduce readability.  Maybe do it for "Consider the examples below." and "The following is an example of a record"